app/src/main/assets/licenses.html

```diff
@@ -786,6 +786,252 @@ <h2 class="inline">9. Accepting Warranty or Additional Liability.</h2>
     </div>
     <div class="library">
       <!-- https://opensource.org/licenses/Apache-2.0 -->
+      <h1 class="title">PhotoView</h1>
```

Comment:
Because `generateLicensePage` task had not done, these differences were occured. 😢